### PR TITLE
Docs: Add light/dark mode screenshot guidelines

### PR DESCRIPTION
**Pull request by imatwawana**

### CODE REVIEW

File: docs/sources/write/image-guidelines/index.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Use the following standards for images and diagrams:
 
 - **Scope**: Limit the contents of an image to the relevant portion.
 
-  Don't include distracting or unnecessary content and whitespace.
+  Don't include distracting or unnecessary content and whitespace, such as your browswer window or tabs, or other window decoration.
```
Comment (imatwawana) [inline docs/sources/write/image-guidelines/index.md:53]:
```suggestion
  Don't include distracting or unnecessary content and whitespace, or your browser window, browser tabs, or other window decoration.
```

**Review by chri2547 — APPROVED** [review]

**Review by clayton-cornell — APPROVED** [review]

**Review by jdbaldry — APPROVED** [review]
Love it! Should we mention it in https://grafana.com/docs/writers-toolkit/whats-new/ (https://github.com/grafana/writers-toolkit/blob/main/docs/sources/whats-new.md?plain=1#L23-L25)?

**Comment by imatwawana** [comment]
> Love it! Should we mention it in [grafana.com/docs/writers-toolkit/whats-new](https://grafana.com/docs/writers-toolkit/whats-new/) ([`main`/docs/sources/whats-new.md?plain=1#L23-L25](https://github.com/grafana/writers-toolkit/blob/main/docs/sources/whats-new.md?plain=1&rgh-link-date=2024-08-29T15%3A58%3A21Z#L23-L25))?

Oh my! Didn't realize we implemented that so fast, lol. Yes, sounds like a plan. I'll update shortly.

File: docs/sources/whats-new.md [hunk 2]
```diff
@@ -24,3 +24,4 @@ This page provides a summary of notable changes to Writers' Toolkit guidance.
 | ------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
 | Revised Vale rule for use of exclamation marks and reduced its Vale severity from "error" to "warning". | [`Grafana.Exclamation`](/docs/writers-toolkit/review/lint-prose/rules/#grafanaexclamation) |
 | Added a What's new page used to summarize notable changes.                                              | [What's new](./)                                                                           |
+| Added guidance regarding dark and light mode for images. | [Media guidelines](https://grafana.com/docs/writers-toolkit/write/image-guidelines/#screenshot-guidelines-1) |
```
Comment (imatwawana) [inline docs/sources/whats-new.md:27]:
```suggestion
| Added guidance regarding dark and light mode for images. | [Media guidelines](/docs/writers-toolkit/write/image-guidelines/#screenshot-guidelines-1) |
```

**Pull request merged**